packages/lib-utils/src/app/redux/reducers/k8s/k8s.ts

```diff
@@ -192,7 +192,8 @@ export const sdkK8sReducer = (state: K8sState, action: K8sAction): K8sState => {
       if (currentJS.toJSON) {
         currentJS = currentJS.toJSON();
         currentJS.metadata.resourceVersion = k8sObjects?.metadata?.resourceVersion;
-        if (isEqual(currentJS, k8sObjects)) {
+        const loadError = state.getIn([action.payload.id, 'loadError']);
```

Comment:
@rottencandy we should use `id` in lieu of `action.payload.id` (already destructured here: https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/lib-utils/src/app/redux/reducers/k8s/k8s.ts#L189)

Stepping back, this seems odd that we are indirectly using this reducer to modify the `loadError` state. This reducer is primarily concerned with handling modified `K8sObjects`.

Would it make more sense to add a new reducer to clear the error? Something like `ActionType.ClearError`? If our goal here is to prevent overriding the state when an error is present, I think we should find where `ActionType.ModifyObject` is dispatched and update to check if there is a `loadError` present before dispatching?

Reply:
Thanks, having a separate reducer to clear the error sounds much better. Updated.